implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Because this class shouldn't be used outside properties library. Static keyword is present for those methods because none non-static field of object is used.